docs/src/app_dev/overview/index.rst

```diff
@@ -8,47 +8,63 @@
 Overview
 ========
 
-An application on the Canton Network is defined by the kind of APIs it integrates with.
-We explain all the APIs available from the Global Synchronizer and a validator node :ref:`here <splice_app_apis>`.
+Canton Network Applications
+---------------------------
 
-At a high-level, we can distinguish between the following two kinds of apps:
+`Canton Network applications <https://docs.digitalasset.com/build/3.3/overview/introduction>`__
+are used to operate business processes spanning
+multiple organizations or business entities.
+Use the following documentation to build Canton Network applications and get them featured on the Canton Network:
 
-* Read-Only Apps: these apps build functionality by using read-only APIs, and thus cannot mutate the ledger.
-* Daml Apps: these apps integrate directly or indirectly with the Ledger API
-  to create new contracts or exercise choices on the ledger to drive on-ledger workflows.
+.. TODO(#1156): link to https://docs.digitalasset.com/build/current/ instead of 3.4 when it is available
 
-You can find more information in the following sub-sections:
+* Learn to build Canton Network applications from the tutorials, how-tos, explanations, and reference documentation at
+  https://docs.digitalasset.com/build/3.3/
+* Browse the currently featured apps: https://sync.global/featured-apps/
+* Request an app to be featured: https://sync.global/featured-app-request/
+* Learn about ongoing CIPs to feature apps from the CIP mailing list (and consider joining the list): https://lists.sync.global/g/cip-discuss/topics
+* Learn about the rationale for featuring an app from the CIP repository: https://github.com/global-synchronizer-foundation/cips
```

Comment:
there are so many cips in there, might wanna link to something specific. I'm not sure which one you mean tbh

Reply:
Good point. I've actually found that there are no CIPs for featuring an app. Let me dig further where the applications are published.